hotsos/core/plugins/openstack/openstack.py

```diff
@@ -262,9 +276,10 @@
                   'keystone': KEYSTONE_EXCEPTIONS,
                   'manila': MANILA_EXCEPTIONS,
                   'masakari': MASAKARI_EXCEPTIONS,
-                  'neutron': NEUTRON_EXCEPTIONS + OVSDBAPP_EXCEPTIONS,
+                  'neutron': NEUTRON_EXCEPTIONS + OVSDBAPP_EXCEPTIONS +
+                  NEUTRONCLIENT_EXCEPTIONS,
                   'nova': NOVA_EXCEPTIONS + PYTHON_LIBVIRT_EXCEPTIONS +
-                  NEUTRONCLIENT_EXCEPTIONS + OS_VIF_EXCEPTIONS,
+                  OS_VIF_EXCEPTIONS,
```

Comment:
i think we need to leave NEUTRONCLIENT_EXCEPTIONS in the nova set since we would expect to find those exceptions in the nova logs but not the neutron logs right?